src/System.Private.Xml/tests/XmlReader/Tests/BaseUriTests.cs

```diff
@@ -28,7 +28,7 @@ public static IEnumerable<object[]> GetXmlReaderUrlCreateMethods()
         [MemberData(nameof(GetXmlReaderUrlCreateMethods))]
         public void CreateWithAbsolutePathGivesAbsoluteBaseUri(Func<string, XmlReader> factory)
         {
-            string tempPath = Path.GetTempFileName();
+            string tempPath = GetTestFilePath();
```

Comment:
You don't need the try/catch or File.Delete at all now; the base class handles deleting the file.